try { 1 div 0 } catch err:* { "Division by zero" }
